### PR TITLE
Simplify types around `expect_events` cheatcode

### PR DESCRIPTION
**Pull request by mkaput**
Just found an opportunity here to clean up things.

### CODE REVIEW

**Review by kasperski95 — APPROVED** [review]
I'd rename `EventExpectation` to `ExpectedEvent`

**Comment by mkaput** [comment]
> I'd rename `EventExpectation*` to `ExpectedEvent*`

@MaksymilianDemitraszek wdyt? We have alternatively thought about the name `EventData`

File: website/docs/tutorials/07-testing/02-cheatcodes/expect-events.md [hunk 1]
```diff
@@ -1,28 +1,15 @@
...
-            )],
-        ) -> None: ...
+class EventExpectationDict(TypedDict):
+    name: str
+    data: NotRequired[list[int] | dict[str, Any]]
+    from_address: NotRequired[int]
+
+EventExpectationName = str
+
+EventExpectation = EventExpectationDict | EventExpectationName
```
Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/07-testing/02-cheatcodes/expect-events.md:10]: I would call it `ExpectedEvent` imo fits better. Expectation suggests behaviour not data

**Review by MaksymilianDemitraszek — CHANGES_REQUESTED** [review]

File: website/docs/tutorials/07-testing/02-cheatcodes/expect-events.md (hunk 1 above)
Comment (mkaput) [reply website/docs/tutorials/07-testing/02-cheatcodes/expect-events.md]: Done

**Comment by mkaput** [comment]
@kasperski95 @MaksymilianDemitraszek actually the name `ExpectedEvent` is already used by something else which is imported in this module... reverting some changes

**Review by MaksymilianDemitraszek — APPROVED** [review]

**Comment by mkaput** [comment]
@kasperski95 ping

**Review by kasperski95 — APPROVED** [review]

**Pull request merged**